### PR TITLE
Removed obsolete python samples

### PR DESCRIPTION
**Pull request by gursimarsingh**
Clean Samples #25006 
This PR removes 36 obsolete python samples from the project, as part of an effort to keep the codebase clean and focused on current best practices. Some of these samples will be updated with latest algorithms or will be combined with other existing samples. 

Removed Samples:

> browse.py
camshift.py
coherence.py
color_histogram.py
contours.py
deconvolution.py
dft.py
dis_opt_flow.py
distrans.py
edge.py
feature_homography.py
find_obj.py
fitline.py
gabor_threads.py
hist.py
houghcircles.py
houghlines.py
inpaint.py
kalman.py
kmeans.py
laplace.py
lk_homography.py
lk_track.py
logpolar.py
mosse.py
mser.py
opt_flow.py
plane_ar.py
squares.py
stitching.py
text_skewness_correction.py
texture_flow.py
turing.py
video_threaded.py
video_v4l2.py
watershed.py

These changes aim to improve the repository's clarity and usability by removing examples that are no longer relevant or have been superseded by more up-to-date techniques.

### CODE REVIEW

**Comment by vpisarev** [comment]
@asmorkalov, I suggest to merge this PR

File: samples/python/dft.py [hunk 1]
```diff
@@ -1,120 +0,0 @@
-#!/usr/bin/env python
```
Comment (asmorkalov) [inline samples/python/dft.py:1]: Make a snippet for dft()?

File: samples/python/distrans.py [hunk 2]
```diff
@@ -1,78 +0,0 @@
...
-    cm = make_cmap('jet')
-    need_update = True
-    voronoi = False
-
-    def update(dummy=None):
-        global need_update
-        need_update = False
-        thrs = cv.getTrackbarPos('threshold', 'distrans')
-        mark = cv.Canny(img, thrs, 3*thrs)
-        dist, labels = cv.distanceTransformWithLabels(~mark, cv.DIST_L2, 5)
```
Comment (asmorkalov) [inline samples/python/distrans.py:44]: Make a snippet for distanceTransform

File: samples/python/fitline.py [hunk 3]
```diff
@@ -1,94 +0,0 @@
...
-    if n > 0:
-        line_points = sample_line(p0, p1, n-outn, noise)
-        outliers = np.random.rand(outn, 2) * (w, h)
-        points = np.vstack([line_points, outliers])
-        for p in line_points:
-            cv.circle(img, toint(p), 2, (255, 255, 255), -1)
-        for p in outliers:
-            cv.circle(img, toint(p), 2, (64, 64, 255), -1)
-        func = getattr(cv, cur_func_name)
-        vx, vy, cx, cy = cv.fitLine(np.float32(points), func, 0, 0.01, 0.01)
```
Comment (asmorkalov) [inline samples/python/fitline.py:68]: Snippet for fitLine

File: samples/python/houghcircles.py [hunk 4]
```diff
@@ -1,49 +0,0 @@
...
-        fn = sys.argv[1]
-    except IndexError:
-        fn = 'board.jpg'
-
-    src = cv.imread(cv.samples.findFile(fn))
-    img = cv.cvtColor(src, cv.COLOR_BGR2GRAY)
-    img = cv.medianBlur(img, 5)
-    cimg = src.copy() # numpy function
-
-    circles = cv.HoughCircles(img, cv.HOUGH_GRADIENT, 1, 10, np.array([]), 100, 30, 1, 30)
```
Comment (asmorkalov) [inline samples/python/houghcircles.py:30]: Snippet for `HoughCircles`

File: samples/python/houghlines.py [hunk 5]
```diff
@@ -1,60 +0,0 @@
...
-    cdst = cv.cvtColor(dst, cv.COLOR_GRAY2BGR)
-
-    if True: # HoughLinesP
-        lines = cv.HoughLinesP(dst, 1, math.pi/180.0, 40, np.array([]), 50, 10)
-        a, b, _c = lines.shape
-        for i in range(a):
-            cv.line(cdst, (lines[i][0][0], lines[i][0][1]), (lines[i][0][2], lines[i][0][3]), (0, 0, 255), 3, cv.LINE_AA)
-
-    else:    # HoughLines
-        lines = cv.HoughLines(dst, 1, math.pi/180.0, 50, np.array([]), 0, 0)
```
Comment (asmorkalov) [inline samples/python/houghlines.py:37]: Snippet for HoughCircles

File: samples/python/kalman.py [hunk 6]
```diff
@@ -1,98 +0,0 @@
...
-
-import numpy as np
-import cv2 as cv
-
-from math import cos, sin, sqrt, pi
-
-def main():
-    img_height = 500
-    img_width = 500
-    kalman = cv.KalmanFilter(2, 1, 0)
```
Comment (asmorkalov) [inline samples/python/kalman.py:27]: snippet for KalmanFilter

File: samples/python/dis_opt_flow.py [hunk 7]
```diff
@@ -1,122 +0,0 @@
-#!/usr/bin/env python
```
Comment (asmorkalov) [inline samples/python/dis_opt_flow.py:1]: Move to snippets

File: samples/python/laplace.py [hunk 8]
```diff
@@ -1,69 +0,0 @@
-#!/usr/bin/env python
```
Comment (asmorkalov) [inline samples/python/laplace.py:1]: Move to snippets.

File: samples/python/lk_track.py [hunk 9]
```diff
@@ -1,106 +0,0 @@
-#!/usr/bin/env python
```
Comment (asmorkalov) [inline samples/python/lk_track.py:1]: Move to snippets for calcOpticalFlowPyrLK.

File: samples/python/mser.py [hunk 10]
```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env python
```
Comment (asmorkalov) [inline samples/python/mser.py:1]: Move to snippets for MSER.

File: samples/python/opt_flow.py [hunk 11]
```diff
@@ -1,104 +0,0 @@
...
-    _ret, prev = cam.read()
-    prevgray = cv.cvtColor(prev, cv.COLOR_BGR2GRAY)
-    show_hsv = False
-    show_glitch = False
-    cur_glitch = prev.copy()
-
-    while True:
-        _ret, img = cam.read()
-        gray = cv.cvtColor(img, cv.COLOR_BGR2GRAY)
-        flow = cv.calcOpticalFlowFarneback(prevgray, gray, None, 0.5, 3, 15, 3, 5, 1.2, 0)
```
Comment (asmorkalov) [inline samples/python/opt_flow.py:76]: Move to snippets for calcOpticalFlowFarneback

File: samples/python/logpolar.py [hunk 12]
```diff
@@ -1,45 +0,0 @@
...
-    except IndexError:
-        fn = 'fruits.jpg'
-
-    img = cv.imread(cv.samples.findFile(fn))
-    if img is None:
-        print('Failed to load image file:', fn)
-        sys.exit(1)
-
-    img2 = cv.logPolar(img, (img.shape[0]/2, img.shape[1]/2), 40, cv.WARP_FILL_OUTLIERS)
-    img3 = cv.linearPolar(img, (img.shape[0]/2, img.shape[1]/2), 40, cv.WARP_FILL_OUTLIERS)
```
Comment (asmorkalov) [inline samples/python/logpolar.py:32]: Move to snippets for `logPolar` and `linearPolar`

File: samples/python/contours.py [hunk 13]
```diff
@@ -1,66 +0,0 @@
-#!/usr/bin/env python
```
Comment (asmorkalov) [inline samples/python/contours.py:1]: Most probably need add snippets for mentioned xxxContours functions.

**Review by asmorkalov — COMMENTED** [review]
Please take a look on decision for corresponding C++ samples and add the same snippets in Python.

File: samples/python/mser.py (hunk 10 above)
Comment (gursimarsingh) [reply samples/python/mser.py:1]: @asmorkalov corresponding cpp sample [detect_mser.cpp](https://github.com/opencv/opencv/blob/5.x/samples/cpp/detect_mser.cpp ) is not in snippets. Should we still move python or keep it same as cpp

File: samples/python/lk_track.py (hunk 9 above)
Comment (gursimarsingh) [reply samples/python/lk_track.py:1]: @asmorkalov corresponding cpp sample [lkdemo.cpp](https://github.com/opencv/opencv/blob/5.x/samples/cpp/lkdemo.cpp ) is not in snippets. Should we still move python or keep it same as cpp

File: samples/python/mser.py (hunk 10 above)
Comment (asmorkalov) [reply samples/python/mser.py:1]: yes, let's move both to snippets.

File: samples/python/texture_flow.py [hunk 14]
```diff
@@ -1,55 +0,0 @@
...
-
-    img = cv.imread(cv.samples.findFile(fn))
-    if img is None:
-        print('Failed to load image file:', fn)
-        sys.exit(1)
-
-    gray = cv.cvtColor(img, cv.COLOR_BGR2GRAY)
-    h, w = img.shape[:2]
-
-    eigen = cv.cornerEigenValsAndVecs(gray, 15, 3)
```
Comment (asmorkalov) [inline samples/python/texture_flow.py:34]: Snippet for `cornerEigenValsAndVecs`

File: samples/python/squares.py [hunk 15]
```diff
@@ -1,55 +0,0 @@
...
-        for thrs in range(0, 255, 26):
-            if thrs == 0:
-                bin = cv.Canny(gray, 0, 50, apertureSize=5)
-                bin = cv.dilate(bin, None)
-            else:
-                _retval, bin = cv.threshold(gray, thrs, 255, cv.THRESH_BINARY)
-            contours, _hierarchy = cv.findContours(bin, cv.RETR_LIST, cv.CHAIN_APPROX_SIMPLE)
-            for cnt in contours:
-                cnt_len = cv.arcLength(cnt, True)
-                cnt = cv.approxPolyDP(cnt, 0.02*cnt_len, True)
```
Comment (asmorkalov) [inline samples/python/squares.py:30]: Snippet for approxPolyDP.

**Comment by asmorkalov** [comment]
CI:
```
/build/precommit_docs/5.x/opencv/modules/imgproc/include/opencv2/imgproc.hpp:4766: warning: included file samples/cpp/contours2.cpp is not found. Check your EXAMPLE_PATH
```

**Review by asmorkalov — APPROVED** [review]

**Review by vpisarev — APPROVED** [review]

**Comment by vpisarev** [comment]
@asmorkalov, I think, it can be merged

**Pull request merged**